packages/bbui/src/Form/Select.svelte

```diff
@@ -21,6 +22,13 @@
     value = e.detail
     dispatch("change", e.detail)
   }
+
+  const extractObjectProperty = (value, property) => {
```

Comment:
Referencing the above comment, we could just remove this function entirely then.